eng/pipelines/source-build-sdk-content-tests.yml

```diff
@@ -0,0 +1,56 @@
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily midnight run
+  branches:
+    include:
+    - main
+    - release/*
+
+pr: none
+trigger: none
+
+pool:
+  name: NetCore1ESPool-Svc-Internal
+  demands: ImageOverride -equals 1es-ubuntu-2004
+
+parameters:
+- name: dotnetDotnetRunID
+  displayName: 'Specific dotnet-dotnet run ID'
+  type: string
+  default: ' '
+
+jobs:
+- template: templates/jobs/sdk-comparison-tests.yml

```

Comment:
FYI, I am proposing we change which legs we run the comparison tests against because it impacts the tests results themselves.  Please don't let this hold up your PR though.  We will make the necessary changes with https://github.com/dotnet/source-build/issues/3292

Reply:
@MichaelSimons YAML has this `foreach` statement - we could potentially put the list in one place as pure data and expand it from there. But might require some YAML-fu so I'd keep it as a follow-up once we have both this and Logan's change in?
What do you think?

Reply:
I have no objection.  I would not hold this PR up for it.  I could see it taking some time to flush out so I would be cautious weighing the ROI.